contracts/MANIFEST.in

```diff
@@ -0,0 +1 @@
+recursive-include src/ *.cairo *.sol
```

Comment:
We also need contract artifacts (ABIs) for both

Reply:
Hmm, actually not sure if we should (does it make sense) add Solidity to PyPI 🤔

Reply:
OpenZeppelin also uses setuptools, maybe we could take some hints from them: https://github.com/OpenZeppelin/cairo-contracts cc @PacoEdwin